src/map/ai/controllers/mob_controller.cpp

```diff
@@ -145,7 +145,7 @@ bool CMobController::CheckDetection(CBattleEntity* PTarget)
 {
     TracyZoneScoped;
     if (CanDetectTarget(PTarget) || CanPursueTarget(PTarget) ||
-        PMob->StatusEffectContainer->HasStatusEffect({ EFFECT_BIND, EFFECT_SLEEP, EFFECT_SLEEP_II, EFFECT_LULLABY }))
+        PMob->StatusEffectContainer->HasStatusEffect({ EFFECT_BIND, EFFECT_SLEEP, EFFECT_SLEEP_II, EFFECT_LULLABY, EFFECT_PETRIFICATION }))
```

Comment:
I don't know this system on retail,  can you link me some proof of this?

Reply:
So after looking into it, that "tapDeaggroTime" seemt to update a mob has "seen" or "known" about an
attacker. 

Checked on retail:
![image](https://user-images.githubusercontent.com/60417494/178133998-736f31a9-3cb1-4296-bcb6-d1fd3c5b35d4.png)
I was standing at 49.5' so I could still see it, I then repeated the test where I moved hundreds of units away and had a second character follow the bee. There's no log because there's no way to prove without a huge packet dump or video of what happened, but I will describe it:

As above, the mob woke up and then went straight for the character that cast Sleep II on it, but at exactly 30 seconds later it deaggroed, waited a couple seconds, then despawned. The bee remained claimed until it stopped moving just before it went unclaimed. This is consistent with `return PMob->CanDeaggro() && (m_Tick >= m_DeaggroTime + 25s);` provided m_DeaggroTime is 5 seconds.